### DISCUSSION TITLE
Assign an individual tester to a build

### DISCUSSION THREAD

**Discussion by dg76** [fastlane/fastlane#17404] [Ideas] [answered]
Hi!

It seems to be possible to assign a group to a TestFlight build:

https://github.com/fastlane/fastlane/issues/12984

Is it also possible to assign an individual tester? Apple provides an API for this ( https://developer.apple.com/documentation/appstoreconnectapi/assign_individual_testers_to_a_build ) but I cannot find a way to add an individual tester using fastlane. Is that already possible?

Thank you very much!

**Comment by rogerluan** [comment, marked as answer]
Hi @dg76 ! 👋 

Unfortunately, that's not possible yet. Let's keep this discussion around so we can track the progress of this feature request 🚀 

Meanwhile, contributions are always welcome 😉 If you feel like diving into _fastlane_, [YourFirstPR.md](https://github.com/fastlane/fastlane/blob/master/YourFirstPR.md) is a great start ❤️ 

Thanks for this suggestion!

**Reply by dg76** [reply]
Ok, I have implemented it now and created a PR here: https://github.com/fastlane/fastlane/pull/17408

Is it good this way? 🙂

**Reply by rogerluan** [reply]
Just saw it! I reviewed your PR and left some comments.
Thanks for opening a PR! ❤️
